test-data/check-modules.yml

```diff
@@ -0,0 +1,17 @@
+# Type checker test cases dealing with modules and imports.
+
+- case: Access imported definitions
+  files:
```

Comment:
Is it possible to omit ``files:`` line? It looks redundant, but I have never worked with YAML.